docs/sources/flow/get-started/install/puppet.md

```diff
@@ -0,0 +1,109 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/get-started/install/puppet/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/get-started/install/puppet/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/get-started/install/puppet/
+- /docs/grafana-cloud/send-data/agent/flow/get-started/install/puppet/
+
+canonical: https://grafana.com/docs/agent/latest/flow/get-started/install/puppet/
+description: Learn how to install Grafana Agent Flow with Puppet
+menuTitle: Puppet
+title: Install Grafana Agent Flow with Puppet
+weight: 560
+---
+
+# Install {{% param "PRODUCT_NAME" %}} with Puppet
+
+You can use Puppet to install and manage {{< param "PRODUCT_NAME" >}}.
+
+## Before you begin
+
+- These steps assume you already have a working [Puppet][] setup.
+- You can add the following manifest to any new or existing module you choose.
+- The manifest installs {{< param "PRODUCT_NAME" >}} from the package repositories. It targets Linux systems from the following families:
+  - Debian (including Ubuntu)
+  - RedHat Enterprise Linux (including Fedora)
+
+## Steps
+
+To add {{% param "PRODUCT_NAME" %}} to a host:
+
+1. Ensure that the following module dependencies are declared and installed:
+    ```json
+    {
+    "name": "puppetlabs/apt",
+    "version_requirement": ">= 4.1.0 <= 7.0.0"
+    },
+    {
+    "name": "puppetlabs/yumrepo_core",
+    "version_requirement": "<= 2.0.0"
+    }
+    ```
+1. Create a new [Puppet][] manifest with the following class to add the Grafana package repositories, install the `grafana-agent-flow` package, and run the service:
+    ```ruby
```

Comment:
```suggestion
    ```

1. Create a new [Puppet][] manifest with the following class to add the Grafana package repositories, install the `grafana-agent-flow` package, and run the service:

    ```ruby
```